lib/libraries/index.js

```diff
@@ -490,3 +507,15 @@ function calculatePercents(matrix, sumArray) {
   }));
   return newValues;
 }
+
+function calculateRange(matrix) {
+  const newValues = matrix.slice(1).map(d => d.map((e, i) => {
+    if (typeof e === 'number') {
+      if (i === 1) {
+        return e - d[i + 1];
+      }
+    }
+    return e;
+  }));
+  return newValues;
+}
```

Comment:
move this function to /lib/dataset/utils/calculate-range

Reply:
add some range chart examples to /test/demo/index + we'll need to update our gh-pages